libr/anal/p/anal_x86_cs.c

```diff
@@ -1013,8 +1013,17 @@ static void anop_esil (RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int len
 		{
 			char *src = getarg (&gop, 1, 0, NULL);
 			char *dst = getarg (&gop, 0, 1, "^");
-			esilprintf (op, "%s,%s,$z,zf,=,$p,pf,=,$s,sf,=,$0,cf,=,$0,of,=",
-				src, dst);
+			char *_dst;
+			if ((a->bits == 64) && dst && (strlen (dst) == 6) && (dst[0] == 'e') && (_dst = strdup (dst))) {
```

Comment:
Too long line. Probably nesting in two lines or moving the logic intoa separate function will be more readable